2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,62 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  temperatures = [];
```

Comment:
The logic looks good to me 👍 
Something to keep in mind: it's a good idea to use `let` or `const` when declaring a variable (like `temperatures`, `city`, `information_string` in your example.)